doc/classes/ItemList.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ItemList" inherits="Control" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Control that provides a list of selectable items (and/or icons) in a single column, or optionally in multiple columns.
+		A vertical list of selectable items.
```

Comment:
I think mentioning columns is important.

(with enough columns you can make it horizontal 🙃)